### DISCUSSION TITLE
How to configure device / change type

### DISCUSSION THREAD

**Discussion by thestumonkey** [make-all/tuya-local#2240] [Q&A]
When I add my device (an outon smart lamp), I get a list of devices it thinks it may be.  However, this is only a fraction of the available devices, and I want to add it as a dreamlight LED strip (not an avaialble option).
Some of the settings are incorrect, and I want to modify them.  I can't find a configure option that doesn't just do it (incorrectly) for me?

**Comment by capitar06** [comment]
I sort of have the same issue. The integration selects a device which has the most resemblance and uses that. I'd like to use some extra DP's extracted from the IoT platform. So I'll be following this discussion as well. :)

**Comment by make-all** [comment]
If you want to tinker at the dps level in your config, then you would be better trying localtuya.

**Reply by kingp0dd** [reply]
i think localtuya doesn't support subdevice (node_id). i'm doing trial and error with my DPS, i hope i can modify or add config files in tuyalocal

**Comment by garry0garry** [comment]
Please tell me how to configure Weather Station correctly? 
https://github.com/make-all/tuya-local/issues/419
